lib/routes/account.js

```diff
@@ -509,13 +510,35 @@ module.exports = function (
           if (shouldSendNewDeviceLoginEmail) {
             // The response doesn't have to wait for this,
             // so we don't return the promise.
-            mailer.sendNewDeviceLoginNotification(
-              emailRecord.email,
-              userAgent.call({
-                acceptLanguage: request.app.acceptLanguage,
-                timestamp: Date.now()
-              }, request.headers['user-agent'])
-            )
+            var location = null
+            var timeZone = null
+            var ip = request.app.clientAddress
```

Comment:
This could probably be moved to the top of the function and reused throughout since it won't be changing in the request.